tests/unit/test_exceptions.py

```diff
@@ -34,58 +29,52 @@ def test_str(self):
 
 
 class TestClientException:
-    def test_inheritance(self):
-        assert isinstance(ClientException(), PRAWException)
-
```

Comment:
I made a mistake in my last review. I didn't realize that the preexisting tests all tested inheritance. I think it's fine to leave all these existing tests in, and it's up to you whether you want to test inheritance on the new exception.